src/framework/ui/select/select.component.tsx

```diff
@@ -243,9 +247,9 @@ class SelectComponent extends React.Component<SelectProps, State> {
   private startIconAnimation = (): void => {
     const { visible } = this.state;
     if (visible) {
-      this.animateIcon(0);
-    } else {
       this.animateIcon(-180);
```

Comment:
Is it unification with the Nebular?

Reply:
It is just better for user experience :) The arrow icon should point down by default and point up when expanded